components/Button.js

```diff
@@ -9,8 +9,14 @@ const StyledButton = styled.button`
   &:hover {
     cursor: pointer;
   }
+  background-color: ${({ $buttonVariant }) =>
+    $buttonVariant === "delete" ? "red" : "#EEDC82"};
```

Comment:
I suggest to avoid color names like `"red"` in favor of hex codes for consistency and better control over design.

```suggestion
  background-color: ${({ $buttonVariant }) =>
    $buttonVariant === "delete" ? "#FF0000" : "#EEDC82"};
```